bin/utils.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+
+const walkSync = require( "walk-sync" );
+const fs = require( "fs" );
+
+function getFilesFromArgs( args ) {
+  let globs = args.slice();
+
+  if ( !globs.length ) {
+    globs.push( "test/*.js" );
```

Comment:
Seems good to do the recursive thing.

For the other extensions:
- Is `.mjs` officially supported yet?
- Can you actually `require` `.jsx` files? I was under the impression they needed to be compiled first.